lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -101,62 +115,128 @@ bool SessionStateEnforcer::m5g_init_session_credit(
   } else {
     session_map[imsi].push_back(std::move(session_state));
   }
-  MLOG(MINFO) << "Added a session (" << session_map[imsi].size()
-              << ") for IMSI " << imsi << " with session context ID ";
+  MLOG(MDEBUG) << "Added a session (" << session_map[imsi].size()
+               << ") for IMSI " << imsi << " with session context ID ";
   return true;
 }
 
-void SessionStateEnforcer::handle_session_init_rule_updates(
+bool SessionStateEnforcer::m5g_update_session_context(
     SessionMap& session_map, const std::string& imsi,
-    SessionState& session_state) {
-  uint32_t l_teid = 0;
-  auto itp        = pdr_map_.equal_range(imsi);
+    std::unique_ptr<SessionState>& session_state,
+    SessionUpdate& session_update) {
+  uint32_t gnode_teid = 0;

```

Comment:
do we have to initialize `gnode_teid` here?

Reply:
Not required here.